autoscheduler/scraper/serializers.py

```diff
@@ -1,38 +1,63 @@
 from rest_framework import serializers
-from .models.course import Course
-from .models.section import Section, Meeting
+from scraper.models import Course, Section, Meeting, Department
 
-class CourseSerializer(serializers.ModelSerializer):
-    """ Serializes a course into an object with information needed by /api/course """
-    class Meta:
-        model = Course
-        fields = ['title', 'credit_hours']
-
-class SectionSerializer(serializers.ModelSerializer):
-    """ Serializes a section into an object with information needed by /api/sections """
-    instructor_name = serializers.SerializerMethodField()
-    meetings = serializers.SerializerMethodField()
+class TermSerializer(serializers.ModelSerializer):
+    """ Serializes a department into an object with information needed by /api/terms """
+    desc = serializers.SerializerMethodField()
 
     class Meta:
-        model = Section
-        fields = ['id', 'crn', 'instructor_name', 'honors', 'meetings',
-                  'section_num', 'web']
+        model = Department
+        fields = ['term', 'desc']
 
-    def get_instructor_name(self, obj): # pylint: disable=no-self-use
-        """ Get the name (id) of this section's instructor.
-            This function is used to compute the value of the instructor_name field.
-        """
-        return obj.instructor.id
 
-    def get_meetings(self, obj): # pylint: disable=no-self-use
-        """ Gets meeting information for this section
-            This function is used to compute the value of the meetings field.
+    def get_desc(self, obj):
```

Comment:
Look at the linting errors in github actions and add lines to silence them (such as `# pylint: disable=no-self-use`

Reply:
The only pylint error I have now that I don't know what to do with is "Too many public methods". I silenced all the other ones that won't automatically be silenced once api/course and api/section are in the files as well.